nsv13/code/modules/overmap/fighters/fighters.dm

```diff
@@ -770,7 +777,7 @@ After going through this checklist, you're ready to go!
 		to_chat(user, "<span class='warning'>[src]'s canopy isn't open.</span>")
 		return
 	if(maint_state < MS_UNSECURE)
-		if(max_passengers > 0 && alert("Enter what seat?",name,"Pilot seat","Passenger seat") !="Passenger seat")
+		if(alert("Enter what seat?",name,"Pilot seat","Passenger seat") == "Pilot seat")

```

Comment:
A cancel button would also be nice.

Reply:
Actually, doesn't this still give both options even for fighters without a passenger seat?

Reply:
It does